app/controllers/reports_controller.rb

```diff
@@ -32,12 +32,15 @@ def set_instance_variables
   end
 
   def data_for_export
-    @data_for_export ||= Exports::TraineeSearchData.new(policy_scope(NewStarterTraineesService.new(census_date(AcademicCycle.current.start_year)).call))
+    @data_for_export ||= Exports::TraineeSearchData.new(
+      policy_scope(FindNewStarterTrainees.new(census_date(AcademicCycle.current.start_year)).call),
+    )
   end
 
   def filename
     current_time = Time.zone.now
-    "#{current_time.strftime('%F_%H_%M_%S')}_New-trainees-#{AcademicCycle.current.start_year}-#{AcademicCycle.current.end_year}-sign-off-Register-trainee-teachers_exported_records.csv"
+    academic_cycle_label = "#{AcademicCycle.current.start_year}-#{AcademicCycle.current.end_year}"
+    "#{current_time.strftime('%F_%H_%M_%S')}-new-starter-#{academic_cycle_label}-report.csv"
```

Comment:
@AbigailMcP Is this file name better or worse? I think it should be shorter anyway. Happy to adjust it.

Reply:
Not sure who decided on the filename tbh! potentially a question for @MaeveRoseveare